src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  color: $color-main;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    @include text-small;
+
+    color: $color-secondary;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+
+    @include content-flex-around;
+
+    align-items: flex-end;
+  }
+
+  &__reviews {
+    @include text-small;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+
+    @include content-flex-around;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $color-secondary;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+    border: none;
+    background-color: $button-color-blue;
+
+    font-size: 14px;
+    font-weight: 700;
+    color: $button-color-white;
+  }
+
+  &:hover &__button {
+    border: 1px solid $button-color-blue;
```

Comment:
It's a good practice to add the border for the default state of the element and just change its color on hover